statsmodels/graphics/tests/test_tsaplots.py

```diff
@@ -51,8 +51,8 @@ def test_plot_month():
     dta = sm.datasets.elnino.load_pandas().data
     dta['YEAR'] = dta.YEAR.astype(int).apply(str)
     dta = dta.set_index('YEAR').T.unstack()
-    dates = lmap(lambda x : pd.datetools.parse('1 '+' '.join(x)),
```

Comment:
By the way, I wasn't able to figure out where/why ``datetools.parse`` was removed from pandas. I searched release notes and the commit log. @jreback ping, just in case this was an accident.